### PR TITLE
enable responsive table

### PR DESCRIPTION
**Pull request by robbymilo**
**What this PR does / why we need it**:

Fixes the table on /installation/helm/reference/.

**Which issue(s) this PR fixes**:
Fixes https://github.com/grafana/website/issues/11218

**Special notes for your reviewer**:

**Checklist**
- [ ] Reviewed the `CONTRIBUTING.md` guide
- [ ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

**Review by jdbaldry — APPROVED** [review]
That's a really cool shortcode to have. Do we have documentation on it? If not I propose that we add it to github.com/grafana/writers-toolkit.

There's one CI failure on linting that seems to be related to Go code so I think we can ignore it. 

There's another CI failure around the Helm docs which apparently need to be regenerated with:

```console
$ make -C docs sources/installation/helm/reference.md
```
https://drone.grafana.net/grafana/loki/18016/5/2

I'm not sure if we need a review from \@grafana/loki-squad to double check that this change is OK, especially given that it affects generated documentation for Helm.

**Review by nate-walters-grafana — APPROVED** [review]
Oh, wow! I didn't even know this existed. Awesome.

**Comment by vlad-diachenko** [comment]
please modify the template file, not the resulting one. 
../production/helm/loki/reference.md.gotmpl

and after this, run the command in `docs` folder : `make sources/installation/helm/reference.md`

**Comment by jeschkies** [comment]
@robbymilo could you generate the documentation so e we can merge this?

File: docs/sources/installation/helm/reference.md [hunk 1]
```diff
@@ -2992,4 +2993,5 @@ null
 		</tr>
 	</tbody>
 </table>
+{{< responsive-table >}}
```
Comment (nate-walters-grafana) [inline docs/sources/installation/helm/reference.md:2996]: Does this need a closing slash?
```suggestion
{{< /responsive-table >}}
```

**Review by nate-walters-grafana — COMMENTED** [review]
L2996 needs a closing slash

**Comment by jdbaldry** [comment]
Hey @vlad-diachenko. I've pushed a commit to update the source template and I've tested locally with `make docs` that you can horizontally scroll the table.

Would you be able give this a merge if you are happy?

cc. @JStickler

**Review by jeschkies — APPROVED** [review]

**Comment by JStickler** [comment]
Do we need to backport this to get the fix to show up in the currently published docs?

**Pull request merged**